lib/YoutubeDLWrapper.py

```diff
@@ -213,14 +213,14 @@ def showMessage(self, msg):
                 util.ERROR('Error in callback. Removing.')
                 _CALLBACK = None
         else:
-            if xbmc.abortRequested:
+            if xbmc.Monitor().abortRequested():
```

Comment:
Let's create in instance of `xbmc.Monitor` in `__init__ ` and use that so we don't create a new instance each time (same for sevice.py). eg.:

```python
def __init__(self, *args, **kwargs):
    self._monitor = xbmc.Monitor()
```

```python
if self._monitor.abortRequested():
```